homeassistant/components/integration/sensor.py

```diff
@@ -106,7 +106,7 @@ def __init__(
         """Initialize the integration sensor."""
         self._sensor_source_id = source_entity
         self._round_digits = round_digits
-        self._state = 0
+        self._state = STATE_UNAVAILABLE
```

Comment:
Isn't `None` to represent unknown state better?

Reply:
STATE_UNAVAILABLE was suggested by @emontnemery

Reply:
None or unavailable is both fine for the statistics. Let's change to none if that's more correct 👍